app/screens/settings/notifications/send_test_notification_notice.tsx

```diff
@@ -0,0 +1,126 @@
+import {View} from 'react-native';
+
+import {sendTestNotification} from '@actions/remote/notifications';
+import SectionNotice from '@components/section_notice';
+import {useServerUrl} from '@context/server';
+import {isMinimumServerVersion} from '@utils/helpers';
+import {logError} from '@utils/log';
+import {tryOpenURL} from '@utils/url';
+
+const TIME_TO_IDLE = 3000;
+
+type Props = {
+    serverVersion: string;
+}
+
+type ButtonState = 'idle'|'sending'|'sent'|'error';
+
+function onGoToNotificationDocumentation() {
+    tryOpenURL('https://docs.mattermost.com/collaborate/mention-people.html');
+}
+
+const styles = {
+    wrapper: {
+        flex: 1,
+        justifyContent: 'flex-end' as const,
+        margin: 16,
+    },
+};
+
+const SendTestNotificationNotice = ({
+    serverVersion,
+}: Props) => {
+    const intl = useIntl();
+    const serverUrl = useServerUrl();
+    const [buttonState, setButtonState] = useState<ButtonState>('idle');
+    const isSending = useRef(false);
+    const timeout = useRef<NodeJS.Timeout>();
+
+    const onSendTestNotificationClick = useCallback(async () => {
+        if (isSending.current) {
+            return;
+        }
+
+        isSending.current = true;
+        setButtonState('sending');
+        const result = await sendTestNotification(serverUrl);
+        if (result.error) {
+            logError(result);
+            setButtonState('error');
+        } else {
+            setButtonState('sent');
+        }
+        timeout.current = setTimeout(() => {
+            isSending.current = false;
+            setButtonState('idle');
+        }, TIME_TO_IDLE);
+    }, []);
+
+    useEffect(() => {
+        return () => {
+            clearTimeout(timeout.current);
+        };
+    }, []);
+
+    const primaryButton = useMemo(() => {
+        let text;
+        let icon;
+        let loading;
+        switch (buttonState) {
+            case 'idle':
+                text = intl.formatMessage({id: 'user_settings.notifications.test_notification.send_button.send', defaultMessage: 'Send a test notification'});
+                break;
+            case 'sending':
+                text = intl.formatMessage({id: 'user_settings.notifications.test_notification.send_button.sending', defaultMessage: 'Sending a test notification'});
+                loading = true;
+                break;
+            case 'sent':
+                text = intl.formatMessage({id: 'user_settings.notifications.test_notification.send_button.sent', defaultMessage: 'Test notification sent'});
+                icon = 'check';
+                break;
+            case 'error':
+                text = intl.formatMessage({id: 'user_settings.notifications.test_notification.send_button.error', defaultMessage: 'Error sending test notification'});
+                icon = 'alert-outline';
+        }
+        return {
+            onClick: onSendTestNotificationClick,
+            text,
+            leadingIcon: icon,
+            loading,
```

Comment:
I may have missed it, but I see `loading` is a boolean, but I don't know how it becomes relevant?  Does that mean the button is disabled, when loading?